app/models/post_analyzer.rb

```diff
@@ -36,7 +37,9 @@ def cook(raw, opts = {})
     result = Oneboxer.apply(cooked) do |url|
       @onebox_urls << url
```

Comment:
Should we keep track of `onebox_urls` if `Oneboxer.cached_onebox` might fail later?

Reply:
`@onebox_urls` keeps all links that should be Oneboxed whether they succeed or not. It is used later to determine the number of links.